tests/integration/test_interface.py

```diff
@@ -171,9 +172,9 @@ def test_column_dtypes(dataset_name: str, request):
     """Tests that column dtypes are as expected"""
     if "TODO" in dataset_name:
         pytest.skip(reason=dataset_name)
-    data = _load_sample_data(dataset_name, request)
     noised_data = request.getfixturevalue(f"noised_sample_data_{dataset_name}")
-    check_noised, _, _ = _get_common_datasets(dataset_name, data, noised_data)
+    idx_cols = IDX_COLS.get(dataset_name)
```

Comment:
This makes sense - why were we previously using `_get_common_datasets`?? (rhetorical question unless you happen to know)

Reply:
It is just a helper function that gets comparable datasets so we can compare unnoised and noised data mainly because of the missing rows that get omitted.